src/BugsnagUnity/SessionTracker.cs

```diff
@@ -43,10 +57,38 @@ public void StartSession()
 
       Client.Send(payload);
     }
+
+    public void StopSession()
+    {
+      var session = _currentSession;
+
+      if (session != null) {
+        session.Stopped = true;
+      }
+    }
+
+    public bool ResumeSession()
+    {
+      var session = _currentSession;
+      var resumed = false;
+
+      if (session == null) {
+        StartSession();
+        resumed = false;
+      } else {
+        resumed = session.Stopped;
+        session.Stopped = false;
+      }
+      return resumed;
+    }
     
     public void AddException(Report report)
     {
-      _currentSession?.AddException(report);
+      var session = _currentSession;
+
+      if (session != null) {
```

Comment:
This null check is the same as `_currentSession?`. [Null operator docs](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/null-conditional-operators)